src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './button.scss'
+
+class Button extends React.Component {
+    
+    render()  {  
+      const buttonClassName = this.props.type === "Primary" ? "primary" : "secondary";
```

Comment:
you can directly create a class with just `type`

```suggestion
      const buttonClassName = `button-${this.props.type}`;
```
 which is 

```
      const buttonClassName = "button-" + this.props.type ;
```

change css classname in `button.scss` accordingly